taccsite_cms/migrations/0001_remove_old_system_monitor_table.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.9 on 2025-01-27 20:33
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    """
+    Removes built-in system monitor plugin in favor of external package.
+
+    The new plugin (`djangocms_tacc_system_monitor`) uses:
+    - The same CMS plugin type name (`TaccsiteSystemMonitorPlugin`)
+    - The same plugin instance data in `cms_cmsplugin`
+
+    NOTE: Due to Django CMS's draft/published page system:
+    - Each plugin instance has two copies in the database.
+    - One is for draft page. One is for published page. Even if both match.
+    - This is normal behavior. This migration handles both copies properly.
```

Comment:
```suggestion
    - This is normal behavior.
```

Given the CAVEAT that existing plugins are lost, I now doubt "This migration handles both copies properly."